resources/views/privacy-interception.blade.php

```diff
@@ -66,16 +66,7 @@
             @endif
 
             <div class="privacy">
-                @php($body = "")
-                @switch(Lang::locale())
-                    @case("de")
-                        @php($body = $agreement->body_md_de)
-                        @break
-                    @case("en")
-                        @php($body = $agreement->body_md_en)
-                        @break
-                @endswitch
-                {!! Markdown::parse($body) !!}
+                {!! Markdown::parse($agreement->{'body_md_'.Lang::locale()}) !!}
```

Comment:
Pledge for revert, weil wir über das Switch-Case besser Sprachumleitungen bauen können (z.B. gibts für Pirate-Speak oder Swedish kein Privacy-Agreement, stattdessen kann normales Englisch angezeigt werden).